setup/sql/dbupdate_04.php

```diff
@@ -21741,3 +21741,7 @@ function writeCtrlClassEntry(ilPluginSlot $slot, array $plugin_data) {
 }
 
 ?>
+<#5251>
```

Comment:
IMO you should use the `5_3_hotfixes.php` file for this update step.

Reply:
Thx for hint. I've changed it.